3.1/docker-entrypoint.sh

```diff
@@ -101,6 +101,8 @@ case "$1" in
 		
 		# ensure the right database adapter is active in the Gemfile.lock
 		cp "Gemfile.lock.${adapter}" Gemfile.lock
+                # ensure to install additional gems for Gemfile.local and plugins
+                bundle echeck || bundle install --without development test
```

Comment:
I can't find any reference to `bundle echeck`, but I did find `bundle check` (https://bundler.io/v1.15/man/bundle-check.1.html); was this perhaps a typo?

This seems generally sane, given that we no longer invoke `bundle install` (#80), and given that it's conditional on whether it's actually deemed necessary.

I think these are also currently indented with spaces, and need to be indented with tabs instead.